frontend/lib/src/dataframes/Quiver.ts

```diff
@@ -1362,6 +1385,7 @@ st.add_rows(my_styler.data)
   }
 
   private static parseFields(schema: ArrowSchema): Record<string, Field> {
+    // Normale data columns are listed first, and all index columns listed last.
```

Comment:
super nit: is this supposed to be `normal`?

Reply:
Updated the comment 👍